packages/components/src/List/List.md

```diff
@@ -2,40 +2,40 @@
+| -------------- | --------------------------------------------------- |
+| getComponent   |  support for the [Inject API](../Inject/Inject.md)  |
+| components     | same. Look below for the possible values            |
+| toolbar        |  ListToolBar component props                        |
+| list           |   spread to the real List                           |
+| id             | html id                                             |
+| displayMode    | oneOf 'large', 'table'                              |
+|  defaultHeight |                                                     |
+| rowHeight      |                                                     |
 
 **ListToolBar Props**
 
-| name | description |
-|----|----|
-| id | html id |
-| actionBar | spread to the ActionBar component |
-| selectAllCheckbox | |
-| display | { onChange } to control the change on display mode |
-| sort | `{ field: 'name', onChange: action(), options: [{ id: 'id', name: 'id'}]`|
-| pagination | `{itemsPerPage: 5, totalResults: 10, onChange: action('pagination.onChange') }`|
-| filter | FilterBar props |
+| name              | description                                                                     |
+| ----------------- | ------------------------------------------------------------------------------- |
+| id                | html id                                                                         |
+| actionBar         | spread to the ActionBar component                                               |
+| selectAllCheckbox |                                                                                 |
+| display           | { onChange } to control the change on display mode                              |
+| sort              | `{ field: 'name', onChange: action(), options: [{ id: 'id', name: 'id'}]`       |
+| pagination        | `{itemsPerPage: 5, totalResults: 10, onChange: action('pagination.onChange') }` |
+| filter            | FilterBar props                                                                 |
 
 **List Props**
 
-| name | description |
-|----|----|
-| columns | Array of {key, label, type} define which columns to display |
-| items | Array of object to display |
-| titleProps | object to control the title cell |
-| itemProps | |
+| name           | description                                                               |
+| -------------- | ------------------------------------------------------------------------- |
+|  columns       |  Array of {key, label, type} define which columns to display              |
+| items          | Array of object to display                                                |
+|  titleProps    |  object to control the title cell                                         |
+| itemProps      |                                                                           |
+| cellDictionary | Array of object to define custom cell renderers or override existing ones |
```

Comment:
It's not an array of object, but an object with key/value, key = renderer name to use in columns props, value = the renderer